gapic/src/main/com/google/gapid/perfetto/models/CpuTrack.java

```diff
@@ -96,8 +101,14 @@ public CpuTrack(QueryEngine qe, CpuInfo.Cpu cpu) {
 
   private ListenableFuture<Data> computeSummary(DataRequest req, Window w) {
     return transform(qe.query(summarySql(w.bucketSize)), result -> {
-      Data data = new Data(req, w.bucketSize, new double[w.getNumberOfBuckets()]);
-      result.forEachRow(($, r) -> data.utilizations[r.getInt(0)] = r.getDouble(1));
+      int len = w.getNumberOfBuckets();
+      String[] concatedIds = new String[len];
+      Arrays.fill(concatedIds, "");
```

Comment:
fill with empty string is redundant I think. The new operator would initialize with empty string I believe.

Reply:
Nevermind.

Reply:
It initialized my array with null sadly, went through a bug fix so that I added this line. T_T

Reply:
Yeah I realized it doesn't happen. My bad